includes/Admin/Cross_Origin_Isolation.php

```diff
@@ -183,9 +183,9 @@ public function style_loader_tag( $tag, $handle, $href ): string {
 	 * @param string $handle The script's registered handle.
 	 * @param string $src    The script's source URL.
 	 *
-	 * @return string
+	 * @return string|mixed
 	 */
-	public function script_loader_tag( $tag, $handle, $src ): string {
+	public function script_loader_tag( $tag, $handle, $src ) {
```

Comment:
Ditto here about possibly retaining return types

Reply:
If you filter `script_loader_tag` to return boolean, that would be passed to `add_attribute`. It would return early like this, 

```if ( $this->starts_with( $url, $site_url ) ) {
			return $html;
		}
```

returning a boolean.